crypto/keyring/keyring.go

```diff
@@ -734,7 +752,7 @@ func newRealPrompt(dir string, buf io.Reader) func(string) (string, error) {
 				continue
 			}
 
-			if err := ioutil.WriteFile(dir+"/keyhash", passwordHash, 0555); err != nil {
+			if err := os.WriteFile(dir+"/keyhash", passwordHash, 0o555); err != nil {
```

Comment:
Might want to suppress this error @ https://github.com/securego/gosec#annotating-code